tweakwcs/tests/test_linearfit.py

```diff
@@ -26,7 +26,7 @@
 }
 
 _ATOL = 10 * _LARGE_SAMPLE_SIZE * np.sqrt(
-    np.finfo(linalg._MAX_LINALG_TYPE).eps
```

Comment:
@mcara since there is a chance `linalg._MAX_LINALG_TYPE` can be `None` in #185 how about switching the commented line to the following in #185:
```
np.finfo(linalg._MAX_LINALG_TYPE or np.float64).eps
```